packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx

```diff
@@ -37,6 +37,19 @@ const createTestEnvironment = (
 });
 
 const testUser = UserMock.random().buildGraphql<TUserGraphql>();
+const testSSOUser = {
+  ...UserMock.random().buildGraphql<TUserGraphql>(),
+  idTokenUserInfo: {
+    iss: '',
+    sub: '',
+    aud: '',
```

Comment:
Nit: let's put some random values here for completeness (as these fields are always defined).

Reply:
Not sure what a good set of values would be, but I updated them in this commit: 0138b05933457a87daefeace91129884567af75a